### ISSUE TITLE
[DOC] differentiate between K8s Node and Longhorn Node

### ISSUE THREAD

**Issue by guettli** [longhorn/longhorn#9782]
## What's the document you plan to update? Why? Please describe

The term "node" gets used for K8s and for Longhorn. 

In the docs it is sometimes not clear (at least if you are new to Longhorn) which meaning the word "node" has.

It would be nice to make this more explicit.

For example in this place: https://longhorn.io/docs/1.7.2/nodes-and-volumes/nodes/default-disk-and-node-config/#customizing-default-disks-for-new-nodes

Where should I set the label: On the K8s node or the Longhorn node?

**Comment by derekbit** [comment]
@guettli Would you like to contribute the doc update?

**Comment by guettli** [comment]
@derekbit yes, if you tell me which kind of "node" the text refers to. At least in the above link I am unsure.
